docs/compatibility-matrix.md

```diff
@@ -0,0 +1,23 @@
+# Rancher2 Provider
+
+### Version compatibility matrix
+
+The version matrix specifies the Terraform provider version _recommended_ to use with the associated minor Rancher version that it was released for. When updating the version matrix, add a row for each Terraform version released with a minor Rancher release.
+
+#### Rancher 2.6
+
+| Terraform provider version | Rancher |
+|----------------------------------------|:-------:|
+| 2.0.0                                  | 2.6.11  |
+| 2.0.1 (2.1.0 for features)             | 2.6.x   |
```

Comment:
Recommending to not list not-yet-released versions, only the ones that we already released. I understand it's an example, but don't want it to be misleading. The same applies to the 2.7 table below.

Reply:
Do you mean to remove the entire second row here or just modify to list

`2.0.x (2.x.0 for features)`

Reply:
Just remove, we explain the approach to versioning in Readme already. This matrix is for released(to be released) versions.